### ISSUE TITLE
file.bin is open in some program???

### ISSUE THREAD

**Issue by pokemonerdotcom** [ABZB/Pokemon-Forme-Insertion#5]
Removing old compilation file and zero files in range
Preparing to rebuild Forme file order
File C:/Users/Desktop/Modded/a218g\792.binis open in some program, being uploaded by Dropbox, etc.This program will now wait another 120 seconds (already waited for 46).
Will try one more time, waiting for 360 seconds

I don't use any program similar to Dropbox or Mega to sync but I have faced this problem.
Windows 10 - v2.15

**Comment by ABZB** [comment]
This is an unfortunate known issue that I haven't managed to replicate (aside from Dropbox) on my own device.

I am currently working on a solution, it will unfortunately take some time because I'm learning the coding and such that I need as as I go  along

**Comment by pokemonerdotcom** [comment]
> This is an unfortunate known issue that I haven't managed to replicate (aside from Dropbox) on my own device.
> 
> I am currently working on a solution, it, unfortunately, may take some time because I'm learning the coding and such that I need as as I go along

Yeah. Don't worry. How about icon and name? When I added it, pokemon icon and name are missing

**Comment by ABZB** [comment]
They're missing from where?

**Comment by pokemonerdotcom** [comment]
> They're missing from where?

mini icon in Pokemon party and name. If you try to make your Pokemon evolve into a new alt form you added.
It will be missing the name and mini icon.

**Comment by ABZB** [comment]
The forme name is _never_ referenced outside of the Pokedex, and in gen 7 we haven't figured out how to make new formes show up there (IIRC in gen 6 they just show up). The name in the spreadsheet is just there for your own tracking purposes so you can easily tell what is what.

Also, as of now, nobody's figured out how the forme names work in gen VII either (there is a text file with them, but the order doesn't match any known table). As such, I'm focusing on getting everything else working first.

---

The party/box icons - those follow a table that is code.bin, which has limited space for new entries until and unless we figure out enough of the entire code file to be able to expand the table in some way, and I still need to _find_ the table and figure out how it works (if different) in gen 7 (Axyn figured out what I just said for XY). As such, I will not be adding this feature for some time, I fear. Right now new formes just default to using the base forme's icon, which is not _awful_.

---

Pokedex/PC portraits- the static portraits shown in the pokedex and in the Pokemon Center when you heal. I've found the GARC that holds them, but have yet to find the table that matches each image to the respective Pokemon forme. The game defaults to showing Bulbasaur's Portrait, at least in gen VII. Once I find the table and am able to work with it, I will add handling for this as well (I fear that this table is also hiding in code.bin somewhere).

**Comment by pokemonerdotcom** [comment]
> The forme name is _never_ referenced outside of the Pokedex, and in gen 7 we haven't figured out how to make new formes show up there (IIRC in gen 6 they just show up). The name in the spreadsheet is just there for your own tracking purposes so you can easily tell what is what.
> 
> Also, as of now, nobody's figured out how the forme names work in gen VII either (there is a text file with them, but the order doesn't match any known table). As such, I'm focusing on getting everything else working first.
> 
> The party/box icons - those follow a table that is code.bin, which has limited space for new entries until and unless we figure out enough of the entire code file to be able to expand the table in some way, and I still need to _find_ the table and figure out how it works (if different) in gen 7 (Axyn figured out what I just said for XY). As such, I will not be adding this feature for some time, I fear. Right now new formes just default to using the base forme's icon, which is not _awful_.
> 
> Pokedex/PC portraits- the static portraits shown in the pokedex and in the Pokemon Center when you heal. I've found the GARC that holds them, but have yet to find the table that matches each image to the respective Pokemon forme. The game defaults to showing Bulbasaur's Portrait, at least in gen VII. Once I find the table and am able to work with it, I will add handling for this as well (I fear that this table is also hiding in code.bin somewhere).

Yeah right. It's still a Great Tool!
Anyway, thank you for answering all my questions.
Have a good time! Keep it up, bro!
